cradle-core/src/main/java/com/exactpro/cradle/testevents/StoredTestEventBatch.java

```diff
@@ -78,7 +79,7 @@ public StoredTestEventBatch(StoredTestEventId id, String name, String type, Stor
 				
 				Set<StoredMessageId> eventMessages = messages != null ? messages.get(child.getId()) : null;
 				if (eventMessages != null)
-					batchMessages.put(child.getId(), Collections.unmodifiableSet(new HashSet<>(eventMessages)));
+					batchMessages.put(child.getId(), Set.copyOf(eventMessages));
```

Comment:
Why do we use copy instead of unmodifiableSet ?

Reply:
According to code inspection recommendation in IntelliJ IDEA: "Reports java.util.Collections unmodifiable collection calls that can be converted to newer collection factory methods. These can be replaced with e.g. List.of() or Set.of() introduced in Java 9 or List.copyOf() introduced in Java 10."

Reply:
These methods are different by complexity
unmodifiableSet - only wrap source collection
copyOf - optionally creates hash set
I think we should expect instance of collection which isn't modified outside of this method.